docs/dev/plugins/cloudformation/deployment/target-artifact.md

```diff
@@ -0,0 +1,51 @@
+# Target Artifact
+
+The **target artifact** is the ZIP file generated by the successful transformation with TOSCAna. This chapter details the contents the artifact built by the CloudFormation Plugin and explains their role in the deployment.
+
+## Files and folders
+
+**Structure:**
+
+```text
+<target-artifact-name>.zip
+├── readme.html
+└── output/
+    ├── template.yaml
+    └── files/
+    └── scripts/
+        ├── deploy.sh
+        ├── file-upload.sh
+        ├── deploy.sh
+        ├── cleanup.sh
+        └── util/
+```
+
+### Readme
+
+The `readme.html` is the readme explaining the requirements for and usage of the target artifact in order to deploy the CloudFormation template.
+
+### Template
+
+The `output/template.yaml` is the CLoudFormation template generated by the transformation. This is a valid CloudFormation template in the YAML specification. However, it can not be successfully deployed on its own, as it requires the files contained in the `output/files/` directory in order to complete the deployment.
+
+### Files folder
+
+The files contained in the `output/files/` folder are needed for the deployment of the CloudFormation template. This could include files such as implementation artifacts, dependencies etc.
+
+### Scripts Folder
+
+The scripts contained in the `output/scripts/` folder are needed for the deployment of the CloudFormation template generated by the CloudFormation plugin. The commands used in these scripts are supplied by the AWS CLI and require the previous setup said CLI as explained in the [requirements](../../../../user/plugins/cloudformation/requirements.md#aws-cli) chapter.
```

Comment:
Maybe state that `requirements` is from the user documentation not the developer documentation.